src/main/kotlin/com/coop/technologies/kotlinInsightApi/InsightAPI.kt

```diff
@@ -3,31 +3,45 @@ package com.coop.technologies.kotlinInsightApi
 import com.alibaba.fastjson.JSON
 import com.google.gson.JsonParser
 import io.ktor.client.*
+import io.ktor.client.features.*
 import io.ktor.client.request.*
 import io.ktor.client.request.forms.*
+import io.ktor.client.statement.*
 import io.ktor.http.*
 import kotlinx.coroutines.runBlocking
 import org.slf4j.LoggerFactory
-import java.lang.reflect.Field
 import java.net.URLConnection
+import kotlin.reflect.KClass
+import kotlin.reflect.KParameter
 import kotlin.reflect.KProperty1
+import kotlin.reflect.full.declaredMemberProperties
 import kotlin.reflect.full.isSubclassOf
 import kotlin.reflect.full.memberProperties
 import kotlin.reflect.full.primaryConstructor
-import kotlin.reflect.jvm.javaType
+import kotlin.reflect.jvm.jvmErasure
 
 object InsightCloudApi {
```

Comment:
Can we please rename this to just "InsightApi"? The "Cloud" part is very confusing, as we are working with Insight Server.